src/scrabble.js

```diff
@@ -1,5 +1,57 @@
-function scrabble() {
+const aplthebetValue = {
```

Comment:
Nice use of the scoring. This made the code look neat

Reply:
I agree with Carolina, great approach to this